_includes/nav.html

```diff
@@ -72,7 +72,7 @@
             {%- unless child.nav_exclude -%}
             <li class="nav-list-item {% if page.url == child.url or page.parent == child.title %} active{% endif %}">
               {%- if child.has_children -%}
-                <a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a>
+                <a href="#" aria-label="Expand UI components" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a>
```

Comment:
Instead of hard-coding `aria-label` to be "Expand UI components", you'll want to use the actual value of the `child`, i.e. `{{ child.title }}`.
```suggestion
                <a href="#" aria-label="{{child.title}}" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a>
```